web/src/pages/salesforce/index.js

```diff
@@ -0,0 +1,263 @@
+                        promising career in the Salesforce ecosystem. Work <br/> <br/>
+                        together with our team and peers to earn your first <br/> <br/>
+                        Salesforce Certification and learn key tech career skills. <br/>
+                    </p>
+                </Container>
+            </section>
+
+            {/* Upcoming Program Dates Section*/}
+            <section id="upcoming-program-dates">
+                <Container fluid className="background--grey pt-4">
+                    <Row>
+                        <h1 className="title">UPCOMING PROGRAM DATES</h1>
+                    </Row>
+                    <Row className="dates">
+                        <Col className="winter-dates">
+                            <h4 className="fw-bold">WINTER 2023</h4>
+                            <h4 className="fw-bold">2.15.23</h4>
+                            <h4>TO</h4>
+                            <h4 className="fw-bold">05.15.23</h4>
+                        </Col>
+                        <Col className="spring-dates">
+                            <h4 className="fw-bold">SPRING 2023</h4>
+                            <h4 className="fw-bold">03.15.23</h4>
+                            <h4>TO</h4>
+                            <h4 className="fw-bold">06.15.23</h4>
+                        </Col>
+                        <div className="button">
+                            <BrandButton>LET'S CHAT</BrandButton>
+                        </div>
+                    </Row>
+                </Container>
+            </section>
+
+            {/* Why Salesforce Section*/}
+            <section id="why-salesforce">
+                <Container>
+                    <Col className="title">                   
+                        <Title>WHY LEARN SALESFORCE?</Title>
+                    </Col>
+                    <Col className="paragraph">
+                        <p>
+                            Career opportunities in the Salesforce ecosystem are growing rapidly, and offer a low hurdle to entry-level <br />
+                            tech jobs. Salesforce is the world's leading customer relationship management solution, used by <br /> 
+                            organizations from Fortune 500 companies to government agencies and local non-profits. The role of <br />
+                            Salesforce Administrator is a great starting point for a career in the ecosystem, and our Accelerator is a <br />
+                            great place to learn the skills you need to get started.
+                        </p> 
+                    </Col>
+                    <Row>
+                        <Col>
+                            <img className="arrow-container" src="https://d33wubrfki0l68.cloudfront.net/9b04c2c87b639f305e5ee1125caae183bbd21ef3/16a10/assets/img/aincu/adults/arrow-down-left.png" alt="Arrow Image"/>
```

Comment:
Same thing with this image, we need to download this and place it in the codebase